PublicationsRouter/v2/sword-out/pubrouter-xml/pubrouter.xsd

```diff
@@ -0,0 +1,143 @@
+                                <xs:attribute name="url" type="xs:string" use="required"/>
+                                <xs:attribute name="format" type="xs:string" use="required"/>
+                                <xs:attribute name="packaging" type="xs:string"/>
+                            </xs:extension>
+                        </xs:simpleContent>
+                    </xs:complexType>
+                </xs:element>
+                <xs:element name="source">
+                    <xs:complexType>
+                        <xs:simpleContent>
+                            <xs:extension base="xs:string">
+                                <xs:attribute name="volume" type="xs:string"/>
+                                <xs:attribute name="issue" type="xs:string"/>
+                            </xs:extension>
+                        </xs:simpleContent>
+                    </xs:complexType>
+                </xs:element>
+                <xs:element name="source_id" maxOccurs="unbounded">
+                    <xs:complexType>
+                        <xs:simpleContent>
+                            <xs:extension base="xs:string">
+                                <xs:attribute name="type" type="xs:string"/>
+                            </xs:extension>
+                        </xs:simpleContent>
+                    </xs:complexType>
+                </xs:element>
+                <xs:element ref="dcterms:publisher" minOccurs="0" maxOccurs="unbounded"/>
+                <xs:element ref="dcterms:format" minOccurs="0" maxOccurs="unbounded" />
+                <xs:element ref="dcterms:title"/>
+                <xs:element ref="rioxxterms:type" minOccurs="0"/>
+                <xs:element ref="dcterms:type" minOccurs="0"/>
+                <xs:element ref="rioxxterms:version" minOccurs="0"/>
+                <xs:element name="start_page" minOccurs="0" type="xs:string"/>
+                <xs:element name="end_page" minOccurs="0" type="xs:string"/>
+                <xs:element name="page_range" minOccurs="0" type="xs:string"/>
+                <xs:element name="num_pages" minOccurs="0" type="xs:integer"/>
+                <xs:element ref="dcterms:language" minOccurs="0" maxOccurs="unbounded"/>
+                <xs:element ref="dcterms:abstract" minOccurs="0"/>
+                <xs:element name="identifier" maxOccurs="unbounded">
+                    <xs:complexType>
+                        <xs:simpleContent>
+                            <xs:extension base="xs:string">
+                                <xs:attribute name="type" type="xs:string" use="required"/>
+                            </xs:extension>
+                        </xs:simpleContent>
+                    </xs:complexType>
+                </xs:element>
+                <xs:element ref="rioxxterms:version-of-record" minOccurs="0"/>
+                <xs:element ref="dcterms:subject" minOccurs="0" maxOccurs="unbounded"/>
+                <xs:element ref="dcterms:dateAccepted" minOccurs="0"/>
```

Comment:
It is a ref, it will reference whatever dcterms:dateAccepted has as it's own definitions.